docs/SPIRVRepresentationInLLVM.rst

```diff
@@ -483,6 +483,15 @@ extra operands ``i32 1`` and ``i32 2``.
 decorates the argument ``b`` of ``k`` with ``Restrict`` in SPIR-V while not
 adding any decoration to argument ``a``.
 
+Loop controls and loop metadata
```

Comment:
Please add at least one example of llvm code, as it's done in the previous section (that is "Function parameter, instruction and global variable decoration through metadata") to add the context and make the section more clear. Thank you.